install/tools/istio_vm_common.sh

```diff
@@ -209,12 +210,13 @@ function istioBootstrapVM() {
 
 # Helper functions for the main script
 
-# If Istio was built from source, copy the artifcats to the current directory, for use
-# by istioProvisionVM
+# If Istio source is available, built from source and copy the artifacts to the current directory, for use
+# by istioBootstrapVM
 function istioCopyBuildFiles() {
   local ISTIO_IO=${ISTIO_BASE:-${GOPATH:-$HOME/go}}/src/istio.io
-  local ISTIO_FILES=${ISTIO_FILES:-.}
+  local ISTIO_STAGING=${ISTIO_STAGING:-$ISTIO_IO/istio/bazel-bin/vm}
 
+  mkdir -p $ISTIO_STAGING
```

Comment:
is source required?  The comment isn't clear and the code seems to indicate yes.

Reply:
Yes, this builds the .deb files from source. Will go away once we test the download of
release .deb.

Reply:
Removed this part, download from release artifacts.